release.sh

```diff
@@ -71,7 +71,14 @@ then
   exit 1
 fi
 
-echo Copying over base Dockerfiles
+printf "\n\nDeleting old container images\n\n"
+
+until docker images | grep "containers.schibsted.io/finntech/node" | awk '{print $3}' | sort | uniq | xargs docker rmi -f
+do
+  true
+done
```

Comment:
`xargs --no-run-if-empty` is the default on mac (BSD), but unfortunately it throws if passed in... http://unix.stackexchange.com/a/61823/175589

Went with a slightly verbose version (but it works better), WDYT?